### PR TITLE
feat: `imageSmoothingEnabled` flag now exposed

### PR DESCRIPTION
**Pull request by srijan-paul**
## Expose the `imageSmoothingEnabled` flag from the Canvas API.

- `imageSmoothingEnabled ctx false` is equivalent to `ctx.imageSmoothingEnabled = false`.

---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [x] Linked any existing issues or proposals that this pull request should close
- [x] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

## Question

I also wanted to add some other calls, like [imageSmoothingQuality](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/imageSmoothingQuality) and [lineDashOffset](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset). Should I add those to this same PR?

### CODE REVIEW

File: src/Graphics/Canvas.js [hunk 1]
```diff
@@ -593,3 +593,9 @@ export function bezierCurveTo(ctx) {
     };
   };
 }
+
+export function imageSmoothingEnabled(ctx) {
+  return function(value) {
+    ctx.imageSmoothingEnabled = value;
```
Comment (sigma-andex) [inline src/Graphics/Canvas.js]: I think you are missing a 
```
    return function() {
```
to make it return an `Effect`

**Review by sigma-andex — CHANGES_REQUESTED** [review]

File: src/Graphics/Canvas.js (hunk 1 above)
Comment (srijanpaul-deepsource) [reply src/Graphics/Canvas.js]: You're right! thanks.

File: pnpm-lock.yaml
Comment (f-f) [inline pnpm-lock.yaml:1]: Could you remove this file? We generally don't use pnpm on these repos

**Review by f-f — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml
Comment (srijan-paul) [reply pnpm-lock.yaml:1]: Pushed it by mistake. Removed.

File: src/Graphics/Canvas.js (hunk 1 above)
Comment (srijan-paul) [reply src/Graphics/Canvas.js]: Ah, right. thanks.

**Review by sigma-andex — APPROVED** [review]
LGTM

**Pull request merged**